core/lib/dal/src/blocks_dal.rs

```diff
@@ -2254,7 +2254,7 @@ impl BlocksDal<'_, '_> {
             WHERE
                 number = $1
             "#,
-            miniblock_number.0 as u32
+            miniblock_number.0 as i32
```

Comment:
Side comment: probably we should create a task to switch all the counter types to u64, just so that we don't have our internal y2k38. Not urgent (lol), but still.